paf-mvp-operator-express/tests/helpers/integration.helpers.ts

```diff
@@ -1,8 +1,33 @@
 import { NodeError, NodeErrorType } from '@core/errors';
 import request from 'supertest';
+import { fromDataToObject, QSParam } from '@core/query-string';
+import { RedirectErrorResponse } from '@core/model/model';
+import { Error, ResponseCode } from '@core/model';
 
-export const assertError = (response: request.Response, status: number, type: NodeErrorType) => {
+export const getRedirectUrl = (response: request.Response) => {
+  return new URL(response.header['location']);
+};
+
+export const removeQueryString = (url: URL) => `${url.protocol}//${url.host}${url.pathname}`;
+
+export const assertRestError = (response: request.Response, status: number, type: NodeErrorType) => {
   expect(response.status).toEqual(status);
   const error = response.body as NodeError;
   expect(error.type).toEqual(type);
 };
+
+export const getRedirectResponse = <T extends { code: ResponseCode; error?: Error | NodeError }>(
```

Comment:
Does it make sense to avoid the generic here and use simply `RedirectErrorResponse`?

Reply:
Ok.